js/app.js

```diff
@@ -1,41 +1,49 @@
-// --- API KEY: AIzaSyCKNNUXyaTDWJp14VhcwraU-mY7o4A27HY --- //
+// Global Variables
+var tubeKey = 'AIzaSyCKNNUXyaTDWJp14VhcwraU-mY7o4A27HY';
+var tubeUrl = 'https://www.googleapis.com/youtube/v3/search';
 
-// --- The endpoint: "https://www.googleapis.com/youtube/v3/search" --- //
-
-$(function(){
-  $('#search-button').on("click", function(event){
-    var searchTerm = $('#query').val();
-    $('#top_movies .clearfix h2').text(searchTerm);
-    getRequest(searchTerm);
-  });
+// --- Function takes query term, displays it and calls getRequest function with query -- //
+$(function() 
+{
+	$('#search-button').on("click", function(event)
+	{
+		var searchTerm = $('#query').val();
+		// -- Takes search term and renders it on top of where clips are displayed -- //
+		$('#top_movies .clearfix h2').text(searchTerm);
+		getRequest(searchTerm);
+	});
 });
 
-function getRequest(searchTerm){
-	// two parameters: URL + function that will be called
-  $.getJSON('https://www.youtube.com/results?search_query='+ searchTerm, function(data){
-	showResults(data.Search);
-  });
+// -- Function sends request w/ query to API -- //
+function getRequest(searchTerm)
+{
+	$.getJSON(tubeUrl + '?part=snippet&key=' + tubeKey + '&q=' + searchTerm, function(data){
+		console.log((data.items));
+		showResults(data.items);
+	});
 }
 
-function showResults(results){
-  $('#top_movies .wrapper .row').remove()
-  var html = "";
-  var counter = 0;
-  $.each(results, function(index,value){
-    if (counter < 6) {
-      // if (value.Poster != 'N/A') {
-        counter++;
-        html += '<div class="row">'+
-            '<div class="post">'+
-              '<img src="' + value.Poster.replace("https://", "http://") + '"/>'+
-              '<h3 class="title">' + value.Title + '</h3>'+
-              '<p class="post_info">' + value.Year + ' | ' + value.Type + '</p>'+
-            '</div>'+
+// -- Function to display search results on page -- //
+function showResults(results)
+{
+	$('#top_movies .wrapper .row').remove();
+	var html = "";
+	var counter = 0;
+
+	$.each(results, function(index, arrayitem){
+		if (counter < 9) {
+			counter ++;
+			html += '<div class="row">' + '<div class="post">' +
```

Comment:
For long strings with multiple parts like this, consider putting them into an array and calling `.join()` on them.  Like this:

```var myBigHTMLString = [